calculatorApp/src/app/home/home.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'home',
+  templateUrl: './home.component.html',
+  styleUrls: ['./home.component.css']
+})
+export class HomeComponent implements OnInit {
+
+  currentNumber = '0';
+  firstOperand = null;
+  operator = null;
+  waitForSecondNumber = false;
+
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+  // pulls first and second number from calculator then sets currentNumber "display"
+  getNumber(v: string) {
+    console.log(v);
+    if (this.waitForSecondNumber) {
+      this.currentNumber = v;
+      this.waitForSecondNumber = false;
+    } else {
+      this.currentNumber === '0' ? this.currentNumber = v : this.currentNumber += v;
+    }
+  }
+
+  // adds decimal to current number
+  getDecimal() {
+    if (!this.currentNumber.includes('.')) {
+      this.currentNumber += '.';
+    }
+  }
+  // performs the calculation entered using switch statement 
+  doCalculation(op, secondOp) {
```

Comment:
Would there be a way to make this more dynamic?  maybe make a variable that equals the operand?
for ex:
return this.firstOperand op= secondOp

I'm not sure if that would actually work, It just seems like there is a possiblity to shorten this code. (you can tell I'm having trouble finding things to improve, this code is great! )